ethereumetl/jobs/export_blocks_job.py

```diff
@@ -63,7 +62,6 @@ def __init__(
 
         self.block_mapper = EthBlockMapper()
         self.receipt_mapper = EthReceiptMapper()
```

Comment:
Is this receipt mapper expected to still be here?

Reply:
I change the function to be static (linter kept warning than 'self' was not used, no need to create an object)